### ISSUE TITLE
Build failed: Timeout "chat response has no accessibility violations"

### ISSUE THREAD

**Issue by chrmarti** [microsoft/vscode#297236]
Build: https://dev.azure.com/monacotools/a6d41577-0fa3-498e-af22-257312ff0545/_build/results?buildId=410839
Changes: https://github.com/Microsoft/vscode/compare/b204273...eadc9a8

The test timed out:

<img width="1160" height="197" alt="Image" src="https://github.com/user-attachments/assets/2f9a365f-7b00-4da8-bba2-f46a2e25da23" />
